test/ibmq/test_account_client_v2.py

```diff
@@ -145,6 +145,23 @@ def test_backend_properties(self):
         properties = api.backend_properties(backend_name)
         self.assertIsNotNone(properties)
 
+    def test_backend_pulse_defaults(self):
+        """Check the backend pulse defaults of each backend."""
+        api = self._get_client()
+        api_backends = [backend_info for backend_info
+                        in api.list_backends()]
```

Comment:
doesn't `api.list_backends()` already return a list?

Reply:
Yes, I don't know how that slipped past me 😅. Thank you Jessie.